### PR TITLE
Mhv 35381 older messages styling

### PR DESCRIPTION
**Pull request by alex-morgun-1**
## Description
image

## Original issue(s)
[JIRA - SM to VA.gov: Fix styling for Older Messages component](https://vajira.max.gov/browse/MHV-35381)

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by mmoyer-va — APPROVED** [review]

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx [hunk 1]
```diff
@@ -1,22 +1,77 @@
...
+  const handleExpandAll = e => {
+    // prevent fromp expanding/collapsing on Tab key press for accessibility
+    if (e.key !== 'Tab') {
+      setIsExpandedAll(!isExpandedAll);
+    }
+  };
+
+  return (
+    <div className="older-messages">
+      <div
```
Comment (alexandec) [inline src/applications/mhv/secure-messaging/components/OlderMessages.jsx:28]: Consider adding `aria-expanded`/`aria-controls` here

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx [hunk 2]
```diff
@@ -1,22 +1,77 @@
...
+          ) : (
+            <i className="fas fa-angle-down fa-lg" />
+          )}
+        </span>
+      </div>
+
+      {isExpanded && (
+        <div>
+          <div className="vads-l-row vads-u-justify-content--flex-end">
+            <span
```
Comment (alexandec) [inline src/applications/mhv/secure-messaging/components/OlderMessages.jsx:51]: See above

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx [hunk 3]
```diff
@@ -3,20 +1,77 @@
...
+        tabIndex={0}
+        onKeyDown={e => {
+          handleExpandOlder(e);
+        }}
+      >
+        <span className="vads-u-font-weight--bold">
+          Older messages in this conversation
+          {isExpanded ? (
+            <i className="fas fa-angle-up fa-lg" />
+          ) : (
```
[the remark this answers is not in the document]
Comment (alexandec) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:42]: Consider using `aria-hidden` here

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx [hunk 4]
```diff
@@ -1,22 +1,77 @@
...
+        role="button"
+        tabIndex={0}
+        onKeyDown={e => {
+          handleExpandOlder(e);
+        }}
+      >
+        <span className="vads-u-font-weight--bold">
+          Older messages in this conversation
+          {isExpanded ? (
+            <i className="fas fa-angle-up fa-lg" />
```
[the remark this answers is not in the document]
Comment (alexandec) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:41]: Consider using `aria-hidden` here

File: src/applications/mhv/secure-messaging/sass/secure-messaging.scss [hunk 5]
```diff
@@ -145,6 +152,33 @@
...
+
+  a:visited {
+    color: $color-link-default;
+    text-decoration: underline;
+    transition-duration: 0.3s;
+    transition-timing-function: ease-in-out;
+    transition-property: color, background-color, border-color;
+  }
+  i {
+    margin: 0.5rem;
```
Comment (alexandec) [inline src/applications/mhv/secure-messaging/sass/secure-messaging.scss:166]: Consider using a design system class instead. Same thing for these other selectors

**Review by alexandec — APPROVED** [review]

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx [hunk 6]
```diff
@@ -3,20 +1,77 @@
...
+              }}
+              role="button"
+              tabIndex={0}
+              className="expand-all-messages-btn vads-u-display--flex vads-u-align-items--flex-start"
+              onKeyDown={e => {
+                handleExpandAll(e);
+              }}
+            >
+              Expand All Messages
+              {isExpandedAll ? (
```
[the remark this answers is not in the document]
Comment (alexandec) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:63]: See above

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx [hunk 7]
```diff
@@ -3,20 +1,77 @@
...
+                <i className="fas fa-angle-up" />
+              ) : (
```
[the remark this answers is not in the document]
Comment (alexandec) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:65]: See above

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (oleksii-morgun-va) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx]: applied. Thank you!

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (oleksii-morgun-va) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:47]: done

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (oleksii-morgun-va) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:75]: done

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx (hunk 1 above)
Comment (oleksii-morgun-va) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:28]: added as well. Thank you for pointing that out. New to aria and accessibility attributes, pardon any inconsistencies

File: src/applications/mhv/secure-messaging/components/OlderMessages.jsx (hunk 2 above)
Comment (oleksii-morgun-va) [reply src/applications/mhv/secure-messaging/components/OlderMessages.jsx:59]: done

File: src/applications/mhv/secure-messaging/sass/secure-messaging.scss (hunk 5 above)
Comment (oleksii-morgun-va) [reply src/applications/mhv/secure-messaging/sass/secure-messaging.scss]: refactored where applicable. some designs we know for sure will change, so will keep that in mind for future edits

**Review by mmoyer-va — APPROVED** [review]

**Pull request merged**